src/sdk/http/http-client.ts

```diff
@@ -1,10 +1,9 @@
 import { URL } from 'url';
-import { Readable } from 'stream';
 
 const API_VERSION = 'v1';
 
 export interface IHttpClient {
-	get(path: string, params?: any): Promise<any>;
+	get(path: string, params?: any, csvResponse?: boolean): Promise<any>;
```

Comment:
I would remove this parameter and update below to only parse the JSON if `content-type: application/json`.